daprdocs/content/en/reference/components-reference/supported-pubsub/setup-rabbitmq.md

```diff
@@ -451,6 +451,28 @@ You can set a time-to-live (TTL) value at either the message or component level.
 If you set both component-level and message-level TTL, the default component-level TTL is ignored in favor of the message-level TTL.
 {{% /alert %}}
 
+## Single Active Consumer
```

Comment:
This needs to be added to the overall example and the spec table at the top of the page, as well.

Reply:
The spec table describe metadata for the rabbitmq pubsub **Component** while the `singleActiveConsumer` flag is set at the **Subscription** level (like `queueName`, `routingKey`, ...). There is no spec for the **Subscription**, should we create one ?

Reply:
There is a spec for subscription here. https://docs.dapr.io/reference/resource-specs/subscription-schema/
Is this where this should be added?

As this is a general subscription wide property it needs a good desription of what is does, and examples of best practices of when to use it. The description below should be changed to describe from a user perspective of why this should be set (rather than from Dapr's perspective), but if there are conditions that should be set, please describe these. For example I do not find this clear, so could you expand this some more "To allow another application instance to take over in case of failure, Dapr must [probe the application's health]({{< ref "app-health.md" >}}) and unsubscribe from pub/sub component.

Reply:
> There is a spec for subscription here. https://docs.dapr.io/reference/resource-specs/subscription-schema/ Is this where this should be added?
> 

I think that makes sense... @DropSnorz does that sound like a good place to put it?

Reply:
The [subscription schema]( https://docs.dapr.io/reference/resource-specs/subscription-schema/) seems to be very generic and do not contains metadata for all pub/sub vendors. I wouldn't add the `singleActiveConsumer` here as it's very RabbitMq specific for edge use cases support.

I've psuhed some details regarding why a user would use this queue property with Dapr and a short note mentioning the drawbacks.

Reply:
Ok. Makes sense. Let's keep this to the RabbitMQ docs.